packages/ui-react/src/components/Button/Button.tsx

```diff
@@ -64,7 +62,7 @@ const Button: React.FC<Props> = ({
   const content = (
     <>
       {startIcon && <div className={styles.startIcon}>{startIcon}</div>}
-      {<span className={classNames(styles.buttonLabel, { [styles.hidden]: busy }) || undefined}>{label}</span>}
+      {<span className={classNames({ [styles.hidden]: busy }) || undefined}>{label}</span>}
       {children}
       {busy && <Spinner className={styles.centerAbsolute} size={'small'} />}
```

Comment:
Should we add an aria-label to the Spinner to tell the screen reader about the busy state?